### PR TITLE
Add Render deployment guide

### PR DESCRIPTION
**Pull request by jess-render**
## About

This PR adds a Render deployment guide for the Prisma ORM. 
It pairs with the example code in https://github.com/prisma/prisma-examples/pull/6990

### CODE REVIEW

File: content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: 'Deploy to Render'
+metaTitle: 'Deploy a Prisma app to Render'
+metaDescription: 'Learn how to deploy a Node.js server that uses Prisma ORM to Render.'
+---
+
+<TopBlock>
```
Comment (jharrell) [inline content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx]: This is a deprecated component. not needed anymore 😊 
```suggestion
```

File: content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+The [Prisma Render deployment example](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/render) contains an Express.js application with REST endpoints and a simple frontend. This app uses Prisma Client to fetch, create, and delete records from its database.
+
+## About Render
+
+[Render](https://render.com) is a cloud application platform that lets developers easily deploy and scale full-stack applications. For this example, it's helpful to know:
+- Render lets you deploy long-running, "serverful" full-stack applications. You can configure Render services to [autoscale](https://docs.render.com/scaling) based on CPU and/or memory usage.
+- Render natively supports [common runtimes](https://docs.render.com/language-support), including Node.js and Bun. In this guide, we'll use the Node.js runtime.
+- Render [integrates with Git repos](https://docs.render.com/github) for automatic deployments upon commits. You can deploy to Render from GitHub, GitLab, or Bitbucket. In this guide, we'll deploy from a Git repository.
+
+</TopBlock>
```
Comment (jharrell) [inline content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx]:
```suggestion
```

File: content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+## Prerequisites
+
+- Sign up for a [Render](https://render.com) account
+
+## Get the example code
+Download the [example code](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/render) to your local machine.
```
Comment (jharrell) [inline content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx:25]:
````suggestion
Download the [example code](https://github.com/prisma/prisma-examples/tree/latest/deployment-platforms/render) to your local machine.
```terminal
curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/deployment-platforms/render
cd render
```
````

File: content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+The logic for the Express app is in two files:
+- `src/index.js`: The API. The endpoints use Prisma Client to fetch, create, and delete data from the database.
+- `public/index.html`: The web frontend. The frontend calls a few of the API endpoints.
+
+### Prisma schema and migrations
+
+The Prisma components of this app are in two files:
+- `prisma/schema.prisma`: The data model of this app. This example defines two models, `User` and `Post`. The format of this file follows the [Prisma schema](https://www.prisma.io/docs/orm/prisma-schema/overview).
+- `prisma/migrations/<migration name>/migration.sql`: The SQL commands that construct this schema in a PostgreSQL database. You can auto-generate migration files like this one by running [`prisma migrate dev`](https://www.prisma.io/docs/orm/prisma-migrate/understanding-prisma-migrate/mental-model#what-is-prisma-migrate).
```
Comment (jharrell) [inline content/200-orm/200-prisma-client/500-deployment/101-traditional/225-deploy-to-render.mdx]:
```suggestion
- `prisma/schema.prisma`: The data model of this app. This example defines two models, `User` and `Post`. The format of this file follows the [Prisma schema](/orm/prisma-schema/overview).
- `prisma/migrations/<migration name>/migration.sql`: The SQL commands that construct this schema in a PostgreSQL database. You can auto-generate migration files like this one by running [`prisma migrate dev`](/orm/prisma-migrate/understanding-prisma-migrate/mental-model#what-is-prisma-migrate).
```

**Review by jharrell — COMMENTED** [review]
Few small fixes needed. 

I'll make sure this is merged after the example is (needed for our dead link checker to work.)

**Comment by jess-render** [comment]
Diffs since your review:
* Updated the build command, based on changes here: https://github.com/prisma/prisma-examples/pull/6990#discussion_r1830065050
* Added a section about seeding the database, based on changes here https://github.com/prisma/prisma-examples/pull/6990#discussion_r1831104273
* Added Render to the list of traditional servers platforms in "Deployment paradigms", and linked to that page from the new guide.

**Review by jharrell — APPROVED** [review]

**Pull request merged**